base/array.jl

```diff
@@ -71,8 +71,36 @@ Union type of [`Vector{T}`](@ref) and [`Matrix{T}`](@ref).
 """
 const VecOrMat{T} = Union{Vector{T}, Matrix{T}}
 
+"""
+    DenseArray{T, N} <: AbstractArray{T,N}
+
+`N`-dimensional dense array with elements of type `T`, often used to represent
+a mathematical vector, matrix, or higher dimensional tensor.
+The elements of a dense array are stored contiguously in memory.
+"""
+DenseArray
+
+"""
+    DenseVector{T}
+
+One-dimensional [`DenseArray`](@ref) with elements of type `T`, often used to represent
+a mathematical vector. Alias for `DenseArray{T,1}`.
```

Comment:
The "often used to represent a mathematical vector" part seems unnecessary here too.